app/exporter/logic/s3.py

```diff
@@ -0,0 +1,19 @@
+"""
+    This module handles all interaction with S3
+"""
+import uuid
+import boto
+from boto.s3.key import Key
+
+conn = boto.connect_s3()
```

Comment:
Right now, it only connects to s3 to save the file - leaving the user experience more or less intact. Any errors with S3 should be logged and dealt with on a subsequent run. 

That being said the application never needs to connect, esp not on startup.